### ISSUE TITLE
Update ugali dwarf catalogs

### ISSUE THREAD

**Issue by mcnanna** [DarkEnergySurvey/ugali#67]
A few issues with the catalogs:
Laevens 1 is listed in the ExtraClusters catalog, but the ra, dec, and reference are all for Laevnes 2/Triangulum II. Triangulum II already exists in the ExtraDwarfs catalog. I think the solution is to change the ra, dec, and reference for Laevens 1 to actually be for Laevens 1.

Aquarius II is not in any catalog. It probably belongs in ExtraDwarfs.

McConnachie15 has an object named "Indus I ?", which is actually Indus II.

**Comment by kadrlica** [comment]
Making this proposal a bit more concrete...

Changes to `extra_clusters.csv`:
* Update Laevens 1 to correct location and reference
```
Laevens 1,174.0675,-10.8774,1403.6593
```

The following changes would be made to `extra_dwarfs.csv`:
* Add Aquarius II (1605.05338)
* Add arxiv reference for Carina II and III (1801.07279)
* Add Antila II (1811.04082)
* There is an "Indus II" object, so no action is required.
```
Aquarius II,338.4813,-9.3274,1605.05338
Carina II, 114.1066,-57.9991,1801.07279
Carina III,114.6298,-57.8997,1801.07279
Antlia II,143.8868,-36.7673,1811.04082
```

**Comment by kadrlica** [comment]
Ok, I've uploaded a new catalog tarball:
https://github.com/DarkEnergySurvey/ugali/releases/download/v1.7.0/ugali-catalogs.tar.gz

@mcnanna can you confirm that this works? If so, I'll remove the older tarball from the release files.

The files are also available on the DES machines at:
`~kadrlica/.ugali/catalogs/extras/`

**Comment by kadrlica** [comment]
@apace7 can you see if we are missing anything else that you know of?

**Comment by mcnanna** [comment]
Thanks Alex, I'm working on re-running everything through the new mask, but this all looks good to me.
________________________________

**Comment by mcnanna** [comment]
Two problems I found:

- Aquarius II is now in `extra_dwarfs.csv` twice. Maybe I was using an outdated version when I said it was missing? 

- The information for Laevens 3 in `extra_clusters.csv` is incorrect (it's actually Sagittarius II). It should be
`Laevens 3, 316.7262, 14.98, Laevens 3, 1507.07546`

**Comment by kadrlica** [comment]
Thanks, updates made on DES machines. I'll cut a new version for github once everything has settled down.

**Comment by apace7** [comment]
I've added a handful of extra star clusters to the extra_clusters.csv file (I think the tarball should be here). 
[ugali-catalogs.tar.gz](https://github.com/DarkEnergySurvey/ugali/files/3109870/ugali-catalogs.tar.gz)

Most of the objects came from http://arxiv.org/abs/1805.06473 (9 mostly with Gaia DR2).  The others added are Bliss 1, Munoz 1, and Price-Whelan 1.

**Comment by kadrlica** [comment]
Found a non-ascii character (I think in `DES J0111-1341`). I've replaced locally and will propagate into a tarball.

**Comment by kadrlica** [comment]
To be distributed with v1.8.0. Closed by #94.
